### PR TITLE
✨ Adds release tag, date and url in /config

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- Adds new templates by @eofli @drniiken (until we can add all of them in database) 
- Adds to ``v0/config`` release information (if available). For instance:
 ```json
{
  "data": {
    "invitation_required": false,
    "api_version": "0.15.0",
    "app_name": "simcore_service_webserver",
    "vcs_release_tag": "staging_ResistanceIsFutile12",
    "vcs_release_date": "2023-02-10T18:03:35.957601",
    "vcs_release_url": "https://github.com/ITISFoundation/osparc-simcore/releases/tag/staging_ResistanceIsFutile12"
  }
}
 ```
and also in ``/static-frontend-data.json``
```json
{
    "apiVersion": "0.15.0",
    "appName": "simcore_service_webserver",
    "vcsReleaseTag": "staging_ResistanceIsFutile12",
    "vcsReleaseDate": "2023-02-10T18:03:35.957601",
    "vcsReleaseUrl": "https://github.com/ITISFoundation/osparc-simcore/releases/tag/staging_ResistanceIsFutile12",
    # ...
    "stackName": "master-simcore",
    "displayName": "o\u00b2S\u00b2PARC",
}
```

## How to test

```cmd
cd services/web/server
make install-dev
pytest -vv test/unit/isolated
```

## Checklist

- [x] Unit tests for the changes exist: test w/ and w/o ``SIMCORE_`` environs
- [x] Runs in the swarm

### CODE REVIEW

File: services/web/server/src/simcore_service_webserver/application_settings.py [hunk 1]
```diff
@@ -72,9 +73,26 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
 
     # RUNTIME  -----------------------------------------------------------
     # settings defined from environs defined when container runs
-    # NOTE: keep alphabetically if possible
+    #
+    # NOTE: Please keep fields alphabetically if possible
     AIODEBUG_SLOW_DURATION_SECS: float = 0
 
+    # Release information: Passed by the osparc-ops-autodeployer
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/application_settings.py:80]: all three of this fields are passed form the auto deployer, right?

**Review by GitHK — APPROVED** [review]
👍

**Review by mguidon — APPROVED** [review]
Thanks

**Review by odeimaiz — APPROVED** [review]
💪

File: services/web/server/src/simcore_service_webserver/application_settings.py (hunk 1 above)
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/application_settings.py:80]: yes

**Pull request merged**